tests/lifecycle/driver_test.go

```diff
@@ -50,6 +52,9 @@ func TestServices(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
+	if err := crypto.InitializeGlobalCodec(noop.NewCodec()); err != nil {
+		os.Exit(-1)
+	}
```

Comment:
It is something we ever mentioned in https://github.com/Azure/open-service-broker-azure/pull/598. But the PR is still pending.